root/static/scripts/edit/URLCleanup.js

```diff
@@ -1818,6 +1818,11 @@ const CLEANUPS = {
   'secondhandsongs': {
     match: [new RegExp('^(https?://)?([^/]+\\.)?secondhandsongs\\.com/', 'i')],
     type: LINK_TYPES.secondhandsongs,
+    clean: function (url) {
+      url = url.replace(/^(https?:\/\/)?([^/]+\.)?secondhandsongs\.com/, 'https://secondhandsongs.com');
+      url = url.replace(/secondhandsongs\.com\/(\w+)\/([\d+]+)(?:[\/#?-].*)$/, 'secondhandsongs.com/$1/$2');
```

Comment:
* URL should probably match from the beginning using `^https:\/\/`
* A single capturing group could be used for the whole not trimmed part and just have `'$1'` for the replacement string